PRODUCTS.md

```diff
@@ -150,6 +150,35 @@ TRÅDFRI Module (unverified):
 
 The module is easy to take apart, and debug wires can be soldered on without separating the radio / MCU PCB from the carrier PCB. The carrier board is labelled "D-Sensor-A1-01-A-V2.0".
 
+### E1745 (Motion Sensor)
+
+Details:
+
+* Battery: 2x CR2032
+
+TRÅDFRI Module (unverified):
+
+* Time selection button: Connected to tK3
+* Day/Night selection button: Connected to tK1
+* Pairing button: Connected to tK2
+
+* The motion detection dome is connected (one pin directly, one indirectly) to an "E93196B 0940A 60A" labelled 8-pin chip.
```

Comment:
This chip appears to be an Elmos E931.96 low power PIR sensor controller - https://www.elmos.com/english/products/sensor-ics/passive-infrared/pyroelectric-sensor-signal-processor-ssp/e93196.html